customTests/azure_cuda_bandwidth.nhc

```diff
@@ -50,15 +50,15 @@ function check_cuda_bw()
 	lock_clocks="sudo nvidia-smi -lgc 1400"
 	result=$(catch_error "$lock_clocks" "$error_smi")
     if ! catch_error "$lock_clocks" "$error_smi"; then
```

Comment:
Is something weird going on with the spacing here?

Reply:
must be switching from vim to vscode let me take a look

Reply:
addressed